app/src/main/java/com/omen273/crossLingo/GameActivity.kt

```diff
@@ -572,9 +598,17 @@ class GameActivity : AppCompatActivity(), CrosswordView.OnLongPressListener,
         if (clueCount > 0) freeClueTimer.restart()
     }
 
-    override fun onWordSolved() {
+    override fun onWordSolved(word: Crossword.Word) {
         freeClueRestart()
         dimmer.stop()
+        if (ttsEnabled) {
+            TTS.speak(
+                word.cells.joinToString(separator = "", transform = { it.chars }),
+                TextToSpeech.QUEUE_FLUSH,
+                null,
+                hashCode().toString()
+            )
```

Comment:
How is it work in case of 1. closing activity, 2. several solved words

Reply:
1. continue speaking 2. interrupt previous and speak the last one